### PR TITLE
Add spectator list feature

### PR DESCRIPTION
**Pull request by Shemicalx**
These changes add a spectator list to the left of the game-container which shows updates whenever a spectator enters or leaves a room. This way, one can know who and how many other players are watching the game.
Temp3 112px

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
 # Pong-3.0
-Pong 3.0 is colorful and fast-paced approach to the classic pong game. Play online with your friends and overwhelm them with your superior skills and balls!
+
```
Comment (sdegueldre) [inline README.md]: Extra newline

**Review by ryanwinchester — APPROVED** [review]
Looks good to me.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
 > e.g.: `PORT=42069 npm start`
 
-#### The technologies used for this project are : 
- * [Parcel](https://parceljs.org/) for bundling and building
- * [Socket.io](https://socket.io/) for websocket communication
- * [Pixi.js](https://www.pixijs.com/) for graphics
- 
+#### The technologies used for this project are :
+
+- [Parcel](https://parceljs.org/) for bundling and building
```
Comment (sdegueldre) [inline README.md]: It seems you've used some kind of automatic formatting, while the new formatting is fine, there is nothing wrong with the current one, and this pollutes the commit history. Please revert all changes that are only formatting changes.

**Review by sdegueldre — CHANGES_REQUESTED** [review]
Changes look good and the feature is nice, unfortunately it looks like you have some form of autoformatter going on, changing a lot of lines for no benefit. It also seems that your changes are based on an older version of the master branch.

Please revert all formatting only changes and rebase your work on top of the latest version of master. Also, can you squash both of your commits? It looks like the second commit fixes a bug introduced by the first.

Thanks for your time and for your work!

**Comment by sdegueldre** [comment]
Merged at #6. I accidentally pushed master on the PR branch, which automatically closed the PR as merged, removing my ability to push further changes on it.

I have removed all formatting-only changes and split the commit in two (one for the spectator list, and one for the windows compatibility on the --watch flag that wasn't mentioned anywhere)

@Shemicalx thanks for your work!

**Pull request closed without merging**